src/cli/plugin.rs

```diff
@@ -0,0 +1,116 @@
+use crate::serve_session::ServeSession;
+
+use memofs::InMemoryFs;
+use memofs::{Vfs, VfsSnapshot};
+use roblox_install::RobloxStudio;
+use snafu::{ResultExt, Snafu};
+use std::{
+    fs::{self, File},
+    io::{self, BufWriter},
+};
+use structopt::StructOpt;
+
+static PLUGIN_FILE_NAME: &str = concat!("rojo-", env!("CARGO_PKG_VERSION"), ".rbxmx");
+
+#[derive(Debug, StructOpt)]
+enum PluginSubcommand {
+    /// Install the plugin in Roblox plugins folder. If the plugin is already installed, installing
+    /// it again will overwrite the current plugin file.
+    Install,
+    /// Removes the plugin if installed.
+    Uninstall,
+}
+
+/// Install rojo's plugin.
+#[derive(Debug, StructOpt)]
+pub struct PluginCommand {
+    #[structopt(subcommand)]
+    subcommand: PluginSubcommand,
+}
+
+#[derive(Debug, Snafu)]
+pub struct PluginError(Error);
+
+#[derive(Debug, Snafu)]
+enum Error {
+    #[snafu(display("Could not locate Roblox Studio: {}", source.to_string()))]
+    CannotLocateRobloxStudio { source: roblox_install::Error },
+
+    #[snafu(display("{}", source))]
+    Io { source: io::Error },
+}
```

Comment:
I recently refactored errors in master to switch to `anyhow` for CLI commands. I think it made managing errors like this a lot easier.